docs/introduction/understanding-docker.md

```diff
@@ -232,7 +232,7 @@ the file system and a read-write layer is added to the image.
 - **Allocates a network / bridge interface:** Creates a network interface that allows the 
 Docker container to talk to the local host. 
 - **Sets up an IP address:** Finds and attaches an available IP address from a pool. 
-- **Executes a process that you specify:** Runs your application, and; 
+- **Executes a process that you specify:** Runs your application.
```

Comment:
This looks good. All of the sentences following the bolded parts end in a period. This change would be consistent with that

Aside: the block as a whole  is somewhat inconsistent. The bolded parts are all good in that they continue the sentence of "Docker does the following:". The descriptive sentences that follow each bolded part behave differently, some stand alone, some also seem to complete the sentence "Docker does the following".